packages/core/src/animation/internal/animationCurveOwner/AnimationCurveOwner.ts

```diff
@@ -143,6 +135,15 @@ export class AnimationCurveOwner<V extends KeyframeValueType> {
     this._assembler.setTargetValue(this.defaultValue);
   }
 
+  getCurrentValue(out: V): V {
+    if (this._cureType._isReferenceType) {
+      this._cureType._copyValue(this.baseEvaluateData.value, out);
+      return out;
+    } else {
+      return this._assembler.getTargetValue();
```

Comment:
Value type use `getTargetValue()` save the final value maybe cause bug, because target value include before layer additive value!